client/src/shared/components/Dropdown/index.tsx

```diff
@@ -0,0 +1,81 @@
+import React, { useState } from 'react';
+
+import ChevronDown from '@/src/shared/icons/ChevronDown';
+
+interface DropdownProps {
+  options: string[]
+  label: string
+}
```

Comment:
- this interface could be usable in the future. Please do export it.
```suggestion
export interface DropdownProps {
  options: string[]
  label: string
}
```

**could you apply this as well to all existing components**